test/docs/subprocess_test.rb

```diff
@@ -89,4 +89,40 @@ class Memo::Create < Trailblazer::Activity::Railway
     end
   end
 
+  it "subprocess automatically wires all termini of a nested activity" do
```

Comment:
@KamilMilewski  Do we need to document this spec ? If not, we can move it to normal test.

Reply:
@yogeshjain999  I think it could be beneficial to document this behavior. I'll add this do docs soon.